assemble/bin/start-daemon.sh

```diff
@@ -103,6 +103,9 @@ if [[ "$SERVICE" != "tserver" || $NUM_TSERVERS -eq 1 ]]; then
    fi
    echo "Starting $SERVICE on $HOST"
 
+   OUTFILE="${ACCUMULO_LOG_DIR}/${SERVICE}_1_${LOGHOST}.out"
```

Comment:
I think we should leave 86-91 and remove 106-111. 106-111 will add _1 for all services when there is only one tserver. If we do this, then you will get .out and .err per tserver instance and everything else will remain the same.